digitalization/digitalization.txt

```diff
@@ -3,71 +3,71 @@ Skim summary statistics
  n variables: 29 
 
 -- Variable type:character -----------------------------------------------------
- variable missing complete   n min max empty n_unique
-     cbsa       0      381 381   5   5     0      381
+  variable missing complete   n min max empty n_unique
+ cbsa_code       0      381 381   5   5     0      381
 
 -- Variable type:factor --------------------------------------------------------
   variable missing complete   n n_unique                     top_counts ordered
  geography       0      381 381      381 Abi: 1, Akr: 1, Alb: 1, Alb: 1   FALSE
 
 -- Variable type:integer -------------------------------------------------------
- variable missing complete   n      mean        sd    p0   p25   p50   p75    p100
-     area       0      381 381  29694.57  11387.92 10180 19740 29420 39740   49740
-    emp16       0      381 381 274706.77 664053.4      0 43520 83000 2e+05 8229570
-    metro       2      379 381      1         0        1     1     1     1       1
-        x       0      381 381    191       110.13     1    96   191   286     381
-     hist
- <U+2587><U+2587><U+2587><U+2587><U+2587><U+2586><U+2587><U+2586>
- <U+2587><U+2581><U+2581><U+2581><U+2581><U+2581><U+2581><U+2581>
- <U+2581><U+2581><U+2581><U+2587><U+2581><U+2581><U+2581><U+2581>
- <U+2587><U+2587><U+2587><U+2587><U+2587><U+2587><U+2587><U+2587>
+ variable missing complete   n      mean        sd    p0   p25   p50   p75
+     area       0      381 381  29694.57  11387.92 10180 19740 29420 39740
+    emp16       0      381 381 274706.77 664053.4      0 43520 83000 2e+05
+    metro       2      379 381      1         0        1     1     1     1
+        x       0      381 381    191       110.13     1    96   191   286
```

Comment:
I don't think you need to keep this "x" column, which might be auto generated from reading .csv files